railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -21,6 +21,11 @@ source 'https://rubygems.org'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+<%- if options.api? -%>
+# Use ActiveModelSerializers to serialize JSON responses
+gem 'active_model_serializers', '~> 0.10.0.rc1'
```

Comment:
@spastorino does popularity matter?

I think @paulwalker already explained what I wanted to say better, but let me describe what is more specific.

  * AMS has finally seen [fragment cache support](https://github.com/rails-api/active_model_serializers/pull/810) just 3 months ago when jbuilder has support even for [russian-doll caching](https://github.com/rails/jbuilder/issues/144) since 2013.
  * AMS doesn't know about `view_context` which jbuilder does have access to.
  * [A false performance benchmark has been there on the internet](https://kirillplatonov.com/2014/11/04/active_model_serializer_vs_jbuilder/), a more realistic benchmark shows that there's only [about 20% difference between them](https://gist.github.com/yuki24/2f884d9e27c6e8998ad4). What's interesting is that the 20% comes from implicit rendering which oder is always O(1). This means in a real-world app, the difference would be tiny.
  * I really like implementing versioning with variant (take a look at [versioncake](https://github.com/bwillis/versioncake) as an example). WIth AMS, I can't do anything like that.
  * Hard to understand APIs. `0.8`, `0.9` and `0.10` all have different interfaces and it's hard to catch up. jbuilder is much more consistent.
  * I respect the maintainers, but there are 100+ ASM issues when jbuilder only has 4.
  * The fact that jbuilder is mainly used when apps need an API as a side of the server rendered app doesn't necessarily mean you shouldn't use it for API-only apps.

I can come up with a lot of cases AMS doesn't cover, but I can't find any reasons why jbuilder shouldn't be the default. I would like to hear more about real-world examples that jbuilder doesn't work and AWS does better than jbuilder. Although, I'm not interested in hearing opinions like "jbuilder looks weird" or "AMS is more OO-ed". These are not real issues, but just opinions.